stream-chat-android-client/src/main/java/io/getstream/chat/android/client/errors/ChatErrorCode.kt

```diff
@@ -61,14 +61,14 @@ public enum class ChatErrorCode(public val code: Int, public val description: St
     API_KEY_NOT_FOUND(API_KEY_NOT_FOUND_ERROR_CODE, "Api key is not found, verify it if it's correct or was created.");
 
     public companion object {
-        private val authenticationErrors = setOf(
+        private fun authenticationErrors() = setOf(
             AUTHENTICATION_ERROR_CODE,
             TOKEN_EXPIRED_ERROR_CODE,
             TOKEN_NOT_VALID_ERROR_CODE,
             TOKEN_DATE_INCORRECT_ERROR_CODE,
             TOKEN_SIGNATURE_INCORRECT_ERROR_CODE,
         )
 
-        public fun isAuthenticationError(code: Int): Boolean = authenticationErrors.contains(code)
+        public fun isAuthenticationError(code: Int): Boolean = authenticationErrors().contains(code)
```

Comment:
We could replace it with something like this:

```suggestion
        public fun isAuthenticationError(code: Int): Boolean = when (code) {
            AUTHENTICATION_ERROR_CODE,
            TOKEN_EXPIRED_ERROR_CODE,
            TOKEN_NOT_VALID_ERROR_CODE,
            TOKEN_DATE_INCORRECT_ERROR_CODE,
            TOKEN_SIGNATURE_INCORRECT_ERROR_CODE -> true
            else -> false
        }
```